new/PRS13123TaklaEwostatewos.xml

```diff
@@ -0,0 +1,68 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13123TaklaEwostatewos" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Takla ʾEwosṭātewos</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DN"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-08-04">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ተክለ፡ ኤዎስጣቴዎስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Takla ʾEwosṭātewos</persName>
+                    <persName xml:lang="gez" xml:id="n2">ተክለ፡ ኤዎስጣቴዎስ፡ ወልደ፡ ጐቢስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Takla ʾEwosṭātewos walda Gʷabis</persName>
+                    <persName xml:lang="en" type="normalized" corresp="#n2">Takla ʾEwosṭātewos son of Gʷabis</persName>
+                    <persName><roleName type="title">sǝyyuma Dabra Māʿḍo</roleName>Takla ʾEwosṭātewos</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ruler"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit notBefore="1600" notAfter="1670"/>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS13123TaklaEwostatewos" passive="ESmy18"/>
+                    <relation name="saws:isRelatedTo" active="PRS13123TaklaEwostatewos" passive="LOC2320DabraM"/>
+                    </listRelation>
```

Comment:
no entry for the father?

Reply:
I do not know, I would not do it, unless he appears elsewhere.